client/my-sites/stats/site.jsx

```diff
@@ -144,8 +147,29 @@ class StatsSite extends Component {
 		this.props.recordTracksEvent( 'calypso_stats_parsely_banner_click' );
 	};
 
+	buildPrivateSiteBanner() {
+		return (
+			<Banner
+				callToAction={ translate( 'Launch your site' ) }
+				className="stats__private-site-banner"
+				description={ translate(
+					'Changing your site from private to public helps people find you and get more visitors. Don’t worry, you can keep iterating on your site.'
+				) }
+				disableCircle="true"
+				event="calypso_stats_private_site_banner"
+				dismissTemporary="true"
```

Comment:
This isn't specified in the PR description, but we will only temporarily dismiss the banner for the current page view. Is that intentional?

Reply:
That's OK -- please make sure to call it out in the main PR description if we do keep it, though.

Reply:
We've discussed this during our team meeting sync. Due to different use cases such as private p2s, community sites etc, we decided to make the dismiss event not permanent. Once customer dismisses it, they won't see it again. That said, we'll explore a time component to see if we can show it after a certain time(like one month) as a reminder.